lib/foobara/util/constants.rb

```diff
@@ -0,0 +1,7 @@
+module Foobara
+  EMPTY_ARRAY = [].freeze unless const_defined?(:EMPTY_ARRAY)
+
+  module Util
+    EMPTY_ARRAY = Foobara::EMPTY_ARRAY unless const_defined?(:EMPTY_ARRAY)
```

Comment:
I think it's OK to just have Foobara::EMPTY_ARRAY though this likely doesn't hurt anything